src/Sylius/Bundle/SettingsBundle/Resources/config/doctrine/model/Settings.orm.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping">
+
+    <mapped-superclass name="Sylius\Bundle\SettingsBundle\Model\Settings" table="sylius_settings">
+        <id name="id" column="id" type="integer">
+            <generator strategy="AUTO"/>
+        </id>
+
+        <field name="schema" column="schema_name" type="string" nullable="false"/>
```

Comment:
You need to create new reslover always if you have many Settings for one schema. When you allow create many Settings for one schema you don't have to create new class.